src/behapy/visUtils.py

```diff
@@ -0,0 +1,87 @@
+'''
+lydia barnes 
+lydiabarnes@tuta.io
+
+created december 2022
+from crnolan visual masking example script
+
+utilities to identify artefacts in and visualise continuous raw fibre photometry data
+
+'''
+
+import numpy as np
+
+import holoviews as hv
+import holoviews.streams as streams
+from holoviews import opts
+from holoviews.operation.datashader import datashade, dynspread, rasterize
+hv.extension('bokeh')
+
+import datashader as ds
+import panel
+panel.extension(comms='vscode')
+from intervaltree import IntervalTree, Interval
+reject_intervals = IntervalTree()
+
+# basic plot, not interactive
+def visRaw(df):
+    return datashade(hv.Curve((df.index, df)), aggregator=ds.count()).redim(
+        x='time', y=hv.Dimension('F')).opts(width=800)
+
```

Comment:
I'll dig out the style recommendations we generally use, but I think it's to have two blank rows between top-level function definitions? Function and file naming should also be all_lower_with_underscores.

Reply:
have just run autopep8, so spacing should be fixed. camel case is my religion but i can concede </3